python/README.md

```diff
@@ -0,0 +1,32 @@
+# Apache Spark
+
+Spark is a fast and general cluster computing system for Big Data. It provides
```

Comment:
So this ends up in the packaging description in PyPI (once we publish there). For people browsing on PyPI they might not be super familiar with PySpark so it probably makes sense to keep the blurb.